.eslintrc.json

```diff
@@ -18,7 +18,8 @@
     "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
     "react/react-in-jsx-scope": "off",
     "import/no-unresolved": "off",
-    "no-shadow": "off"
+    "no-shadow": "off",
+    "react/prop-types": "off"
   },
```

Comment:
- [ ] Although we have some text as our PR, we are highly encouraged to maintain a professional PR and we can achieve that by list our some tasks from the project description and some key components to highlight in the project. Let us kindly fix it